omnibus/omnibus.rb

```diff
@@ -14,6 +14,10 @@
 # Do not set this environment variable if building locally.
 # This cache is only necessary because Datadog is building
 # the agent over and over again in a highly distributed environment.
```

Comment:
Could you move this comment (which refers to `S3_OMNIBUS_CACHE_BUCKET`) below the code block you added?

Reply:
Ah yes sorry for this!